Resources/views/backend/mollie_extend_payment/view/payment/form_panel.js

```diff
@@ -142,6 +146,43 @@ Ext.define('Shopware.apps.Mollie.Payment.view.payment.FormPanel', {
                         }
                     ]
                 },
+                {
+                    xtype: 'fieldset',
+                    flex: 12,
+                    border: false,
+                    margin: '0',
+                    bodyPadding: 0,
+                    layout: 'hbox',
+                    items: [
+                        {
+                            xtype: 'combobox',
+                            name: 'mollie_banktransfer_flow',
+                            flex: 10,
+                            fieldLabel: me.molSnippets.captionBankTransferFlow,
+                            labelWidth: labelWidth,
+                            supportText: '',
+                            translatable: true,
+                            multiSelect: false,
+                            editable: false,
+                            allowBlank: false,
+                            store: [
+                                [1, me.molSnippets.valueBankTransferFlowMollie],
+                                [2, me.molSnippets.valueBankTransferFlowNoMollie],
+                            ],
+                        },
+                        {
+                            xtype: 'button',
+                            flex: 2,
+                            text: me.molSnippets.methodReadMore,
+                            cls: 'small primary',
+                            margin: '2 0 0 10',
+                            scope: this,
+                            handler: function () {
+                                window.open('https://github.com/mollie/Shopware/wiki/SEPA-Bank-Transfer', '_blank').focus();
```

Comment:
Nice! Fetching Wiki from github. But if you change the wiki because of a newer version, the old versions would get the updated information from github. Is this really ok? Because it could be information that is not correct for the old version

Reply:
this is no fetch, it should just open the wiki page with further instructions on the features. and this will be done in the bank transfer instructions page.
thats a requirement